src/Item.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+import { db } from './lib/firebase';
+
+function Item(props) {
+  const [checked, setChecked] = useState(false);
+
+  useEffect(() => {
+    // setChecked(props.lastPurchaseDate && true);
+    if (props.lastPurchaseDate) {
+      const timeChecked = 86400; //Change this value if you don't want to wait 24 hrs
```

Comment:
Could use a comment here to explain what `86400` represents.

Reply:
Great point! We'll add a comment.